changelogs/fragments/155-add-opnsense-confi-role.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
I suggest rename this file to `changelogs/fragments/155-add-opnsense-config-role.yml`